docs/tutorial.md

```diff
@@ -0,0 +1,815 @@
+    userId: new UUID("6e841e28-b1bc-4719-bf42-ba4b164fc546"),
+    user: 'testUser',
+    db: 'test-db',
+    roles: [ { role: 'readWrite', db: 'test-db' } ],
+    mechanisms: [ 'SCRAM-SHA-1', 'SCRAM-SHA-256' ]
+  }
+]
+```
+
+Feel free to test out any other MongoDB commands. When you’re ready to leave the MongoDB shell, just type {command}`exit`. You will be back in the host of Charmed MongoDB (`mongodb/0`). 
+
+Exit this host by once again typing {command}`exit`. Now you will be in your original shell where you can interact with Juju and LXD. 
+
+```{note}
+If you accidentally exit one more time or close your terminal session at some point, note that all the environment variables used in the URI will be removed. 
+
+If this happens, simply re-export these variables as described in {ref}`generate-the-full-uri`
+```
+
+## Scale your replicas
+
+A replica set in MongoDB is a group of processes that copy stored data in order to make a database highly available. Replication provides redundancy, which means the application can provide self-healing capabilities in case one replica fails. 
+
+```{admonition} Production disclaimer
+:class: warning
+This tutorial hosts all replicas on the same machine. 
+
+**This should never be done in a production environment**. 
+
+To enable high availability in a production environment, replicas should be hosted on different servers to [maintain isolation.](https://canonical.com/blog/database-high-availability)
+```
+
+### Add replicas
+
+You can add two replicas to your deployed MongoDB application with:
+
+```shell
+juju add-unit mongodb -n 2
+```
+
+It usually takes several minutes for the replicas to be added to the replica set. You’ll know that all three replicas are ready when {command}`juju status --watch 1s` reports:
+
+```text
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  3.6.9   unsupported   14:42:04Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      3  mongodb  6/stable  229  no       Replica set primary
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp  Replica set primary
+mongodb/1   active    idle   1        10.23.62.55     27017/tcp  Replica set secondary
+mongodb/2   active    idle   2        10.23.62.243    27017/tcp  Replica set secondary
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  jammy       Running
+1        started  10.23.62.55   juju-d35d30-1  jammy       Running
+2        started  10.23.62.243  juju-d35d30-2  jammy       Running
+```
+
+#### Verify replica set
+If you want to verify the replica set configuration, you could connect to MongoDB via `charmed-mongodb.mongo`. Since your replica set has 2 additional hosts, you will need to update the hosts in your URI. You can retrieve these host IPs with:
```

Comment:
```suggestion
If you want to verify the replica set configuration, you could connect to MongoDB via `charmed-mongodb.mongosh`. Since your replica set has 2 additional hosts, you will need to update the hosts in your URI. You can retrieve these host IPs with:
```